src/config.rs

```diff
@@ -14,6 +19,43 @@ pub struct Config {
     pub relay: Relay,
     pub prometheus: Prometheus,
     pub cors: bool,
+    pub keys: Keys,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq, Eq)]
+#[serde(tag = "type")]
+pub enum Keys {
+    Static(Static),
+}
+
+impl Default for Keys {
+    fn default() -> Self {
+        Self::Static(Default::default())
+    }
+}
+
+#[serde_as]
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq, Eq)]
+pub struct Static {
+    #[serde_as(as = "Base64<UrlSafe, Unpadded>")]
+    secret: Vec<u8>,
+}
+
+impl From<Static> for StaticSecret {
+    fn from(s: Static) -> Self {
+        StaticSecret::new(s.secret)
+    }
+}
+
+impl Default for Static {
+    fn default() -> Self {
+        Self {
+            // bit of a cludge, but we need to impl default :(
+            secret: kepler_core::libp2p::identity::ed25519::SecretKey::generate()
```

Comment:
In my understanding, it just needs to be a random sequence? Not an actual key?

Reply:
correct, I just used this here as it's an easily accessible random function

Reply:
I've removed the default impl, kepler will exit with an error if the secret is not provided via the config